utils/const.go

```diff
@@ -0,0 +1,5 @@
+package utils
+
+const (
+	MB = 1024 * 1024 // 1 Megabyte
```

Comment:
I'd prefer be explicit
```suggestion
	Megabyte = 1024 * 1024
```

Reply:
Initially I thought `MB` is better, but just realised that your suggestion is better to read. Renamed, thanks